claat/serve.go

```diff
@@ -72,214 +60,232 @@ func cmdServe() {
+	"polymerelements/iron-autogrow-textarea":        "1.0.15",
+	"polymerelements/iron-behaviors":                "1.0.17",
+	"polymerelements/iron-checked-element-behavior": "1.0.5",
+	"polymerelements/iron-collapse":                 "1.3.0",
+	"polymerelements/iron-dropdown":                 "1.5.5",
+	"polymerelements/iron-fit-behavior":             "1.2.6",
+	"polymerelements/iron-flex-layout":              "1.3.2",
+	"polymerelements/iron-form-element-behavior":    "1.0.6",
+	"polymerelements/iron-icon":                     "1.0.12",
+	"polymerelements/iron-icons":                    "1.2.0",
+	"polymerelements/iron-iconset-svg":              "1.1.0",
+	"polymerelements/iron-input":                    "1.0.10",
+	"polymerelements/iron-jsonp-library":            "1.0.4",
+	"polymerelements/iron-localstorage":             "1.0.6",
+	"polymerelements/iron-media-query":              "1.0.8",
+	"polymerelements/iron-menu-behavior":            "1.2.0",
+	"polymerelements/iron-meta":                     "1.1.2",
+	"polymerelements/iron-overlay-behavior":         "1.10.3",
+	"polymerelements/iron-pages":                    "1.0.8",
+	"polymerelements/iron-resizable-behavior":       "1.0.5",
+	"polymerelements/iron-selector":                 "1.5.2",
+	"polymerelements/iron-validatable-behavior":     "1.1.1",
+	"polymerelements/neon-animation":                "1.2.4",
+	"polymerelements/paper-behaviors":               "1.0.12",
+	"polymerelements/paper-button":                  "1.0.14",
+	"polymerelements/paper-dialog":                  "1.1.0",
+	"polymerelements/paper-dialog-behavior":         "1.2.7",
+	"polymerelements/paper-drawer-panel":            "1.0.11",
+	"polymerelements/paper-dropdown-menu":           "1.5.0",
+	"polymerelements/paper-fab":                     "1.2.0",
+	"polymerelements/paper-header-panel":            "1.1.7",
+	"polymerelements/paper-icon-button":             "1.1.4",
+	"polymerelements/paper-input":                   "1.1.23",
+	"polymerelements/paper-item":                    "1.2.1",
+	"polymerelements/paper-listbox":                 "1.1.2",
+	"polymerelements/paper-material":                "1.0.6",
+	"polymerelements/paper-menu":                    "1.2.2",
+	"polymerelements/paper-menu-button":             "1.5.2",
+	"polymerelements/paper-radio-button":            "1.3.1",
+	"polymerelements/paper-radio-group":             "1.2.1",
+	"polymerelements/paper-ripple":                  "1.0.9",
+	"polymerelements/paper-scroll-header-panel":     "1.0.16",
+	"polymerelements/paper-styles":                  "1.2.0",
+	"polymerelements/paper-tabs":                    "1.8.0",
+	"polymerelements/paper-toast":                   "1.3.0",
+	"polymerelements/paper-toolbar":                 "1.1.7",
+	"polymerelements/platinum-sw":                   "1.2.4",
+	"polymerelements/sinonjs":                       "1.17.1",
+	"polymerelements/stacky":                        "1.3.2",
+}
+
```

Comment:
Add short comment to say what this function does?

Reply:
Done